src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpRawlsDAO.scala

```diff
@@ -71,12 +71,8 @@ class HttpRawlsDAO( implicit val system: ActorSystem, implicit val executionCont
     allPublishedPipeline(Get(rawlsAdminWorkspaces)) map {response =>
       response.entity.as[Seq[RawlsWorkspace]] match {
         case Right(srw) =>
-          logger.info("admin workspace list got: " + srw.length + " raw workspaces")
-          val published = srw.collect {
-            case rw:RawlsWorkspace if isPublished(rw) => rw
-          }
-          logger.info("admin workspace list collected: " + published.length + " published workspaces")
-          published
+          logger.info("admin workspace list reindexing: " + srw.length + " published workspaces")
```

Comment:
you can remove the //TODO above at line 59 and the isPublished method
just for my own benefit: where is the either defined that we are using to check the results?